extension/src/popup/components/ErrorTracking/index.tsx

```diff
@@ -3,15 +3,15 @@ import * as Sentry from "@sentry/browser";
 import { Integrations } from "@sentry/tracing";
 
 import { settingsDataSharingSelector } from "popup/ducks/settings";
-import { version } from "../../../../package.json";
+import packageJson from "../../../../package.json";
```

Comment:
this started throwing a warning - it didn't like a named export from a json file